app/views/casa_admin_mailer/account_setup.html.erb

```diff
@@ -1,22 +1,49 @@
-<meta itemprop="name" content="Confirm Email" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-<table width="100%" cellpadding="0" cellspacing="0" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-  <tr style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-    <td class="content-block" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
+<style>
+  meta, .account-setup body, .account-setup tr, .account-setup td, .account-setup a {
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 14px;
+    margin: 0;
+  }
+  .account-setup td {
+    padding: 0 0 20px;
+  }
+  .account-setup a {
+    color: #FFF;
+    text-decoration: none;
+    line-height: 2em;
+    font-weight: bold;
+    text-align: center;
+    cursor: pointer;
+    display: inline-block;
+    border-radius: 5px;
+    text-transform: capitalize;
+    background-color: #348eda;
+    border-color: #348eda;
+    border-style: solid;
+    border-width: 10px 20px;
+  }
+</style>
+<meta itemprop="name" content="Confirm Email">
+<table class="account-setup" width="100%" cellpadding="0" cellspacing="0">
```

Comment:
`width="100%" cellpadding="0" cellspacing="0"` these too
see https://stackoverflow.com/questions/339923/set-cellpadding-and-cellspacing-in-css